client/src/constants/validation/schema/schema-create-phase.js

```diff
@@ -2,14 +2,22 @@ import * as yup from 'yup';
 import { validateDateIsReasonable, errorMessage, errorDateIsReasonable } from '../functions';
 
 export const CreatePhaseSchema = yup.object().shape({
-  phaseName: yup.string().required('Phase name is required'),
+  phaseName: yup.string().required('Phase name is required').max(255, 'Phase name is too long'),
   startDate: yup
     .date()
     .required(errorMessage)
     .typeError(errorMessage)
     .test('is-reasonable', errorDateIsReasonable, validateDateIsReasonable),
   endDate: yup
     .date()
+    .when('startDate', (startDate, schema) => {
+      if (startDate) {
+        const nextDay = new Date(startDate.getTime() + 86400000);
+        return schema.min(nextDay, 'End date must be at least 1 day after Start date');
+      } else {
+        return schema;
+      }
+    })
     .required(errorMessage)
     .typeError(errorMessage)
     .test('is-reasonable', errorDateIsReasonable, validateDateIsReasonable),
```

Comment:
`.min(yup.ref('startDate'), "End date can't be before start date")` could have been used instead of the when statement, but it didn't prevent the same day from being selected.